Libplanet.Net/Transports/NetMQTransport.cs

```diff
@@ -689,7 +697,7 @@ private void ReceiveMessage(object sender, NetMQSocketEventArgs e)
             if (_router.TrySendMultipartMessage(TimeSpan.FromSeconds(1), message))
             {
                 _logger.Debug(
-                    "{Message} as a reply to {Identity} sent.", messageType, reqId);
+                    "{Message} as a reply to {Identity} sent.", message, reqId);
```

Comment:
```suggestion
                    "{Message} as a reply to {Identity} sent.", messageType, reqId);
```

I guess it wasn't intended.

Reply:
Amended, Thanks!